server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/PushSubscriptionSetMethodContract.scala

```diff
@@ -820,4 +836,100 @@ trait PushSubscriptionSetMethodContract {
            |    ]
            |}""".stripMargin)
   }
+
+  @Test
+  def setMethodCreateShouldCallVerificationToPushServer(pushServer: ClientAndServer): Unit = {
+    val request: String =
+      s"""{
+         |    "using": ["urn:ietf:params:jmap:core"],
+         |    "methodCalls": [
+         |      [
+         |        "PushSubscription/set",
+         |        {
+         |            "create": {
+         |                "4f29": {
+         |                  "deviceClientId": "a889-ffea-910",
+         |                  "url": "${getPushServerUrl(pushServer)}",
+         |                  "types": ["Mailbox"]
+         |                }
+         |              }
+         |        },
+         |        "c1"
+         |      ]
+         |    ]
+         |  }""".stripMargin
+
+    `given`
+      .body(request)
+    .when
+      .post
+    .`then`
+      .statusCode(SC_OK)
+
+    pushServer.verify(HttpRequest.request()
+      .withPath("/subscribe"),
```

Comment:
Can we verify the format of the message sent?

cf: https://jmap.io/spec-core.html#pushsubscription

```
The server also immediately makes a POST request to https://example.com/push/?device=X8980fc&client=12c6d086 with the data:

{
  "@type": "PushVerification",
  "pushSubscriptionId": "P43dcfa4-1dd4-41ef-9156-2c89b3b19c60",
  "verificationCode": "da1f097b11ca17f06424e30bf02bfa67"
}
```

Reply:
For validate body request, It already at  `org.apache.james.jmap.pushsubscription.PushSubscriptionSetCreateProcessorTest#pushVerificationShouldSuccess` 

In `PushSubscriptionSetMethodContract`, we don't have a guiceProbe (or something) for get "verificationCode".

Reply:
> For validate body request, It already at org.apache.james.jmap.pushsubscription.PushSubscriptionSetCreateProcessorTest#pushVerificationShouldSuccess

I would love to see it in an integration test